### ISSUE TITLE
SMB Broken after Switch install on batcoera v38

### ISSUE THREAD

**Issue by paronowitz** [ordovice/batocera-switch#232]
After installing switch via the "easy method", SMB mount via \\<ip address> on a windows host no longer works. If I manually re-run batocera-upgrade after installing switch whatever the installer broke with SMB is fixed and switch roms still launch and seem to be playable.

**Comment by uureel** [comment]
not sure what you mean by "SMB mount vi \ on a windows host no longer works", can you explain and provide steps to reproduce?

**Comment by paronowitz** [comment]
If I attempt to mount batocera smb share...ie "net use d: \\<battocera ip address it fails to mount the share (how I typically transfer content from my pc to my machine running batocera). I did confirm if after installing switch emulation, I re-run the batocera v38 install from command line ie batocera-upgrade https://batocera.org/upgrades/x86_64/stable/last that the batocera smb service works again. This is only a problem with v38

**Comment by uureel** [comment]
i see, thx; can't reproduce it on any box, works fine, samba has worked fine with the addon since i first stumbled on bato @ v35 
![image](https://github.com/ordovice/batocera-switch/assets/116395185/399069c9-bb95-4922-a41a-23b763268d90)
further, if you reflash/upgrade/downgrade batocera, the addon stuff is still being launched via custom.sh on each boot (doing pretty much the same thing running the installer does); that's why you're experiencing 'switch roms still launch and seem to be playable', so in other words your samba issue doesn't seem nor seems to be able to seem, to be related to the addon; 
i'll keep this issue open if you can pinpoint something i can reproduce

**Comment by ordovice** [comment]
@paronowitz What is the error you get?  I'm unable to reproduce either.  Have you tried adding /user:root linux to your command?

**Comment by paronowitz** [comment]
Heres an example... I can ftp to the batocera instance but I cannot access the smb share from my windows host after installing switch emulation. If I manually reapply v38, I can once again access the smb share.


![error](https://github.com/ordovice/batocera-switch/assets/116745537/9387b436-6f70-4c25-8af0-07aa4ff3c367)
![ftpsuccess](https://github.com/ordovice/batocera-switch/assets/116745537/6b7e28a1-09db-46ca-85ee-4d85b9a50159)

**Comment by ordovice** [comment]
Interesting.  Can you expand see details next?  I've seen issues with credentials causing issues.

**Comment by paronowitz** [comment]
![image](https://github.com/ordovice/batocera-switch/assets/116745537/fc37b196-50ef-4bd0-9563-058d066ad5fc)

**Comment by paronowitz** [comment]
Any idea why that would only happen after installing switch emulation? Ive tested this on a completely new batocera instance with no roms or config changes.... Image an nvme drive with batcera v38 setup ip etc... everything is accessible as you would expect. Run the switch install script and smb is no longer accessible. Re-run batocera-upgrade https://mirrors.o2switch.fr/batocera/x86_64/stable/last/ and it is accessible again

**Comment by uureel** [comment]
also noticed you're using overlay (which the addon doesn't use), and one thing running `batocera-upgrade https://mirrors.o2switch.fr/batocera/x86_64/stable/last/` actually does is it removes the overlay, so, maybe that's worth checking out

**Comment by paronowitz** [comment]
Strange... The only other thing I notices after running the switch install was the OS changed from 38 to 38oc, I just assumed there was some kernel patch involved 
![image](https://github.com/ordovice/batocera-switch/assets/116745537/b572375f-d71e-461f-8ba0-dcb162ac0d37)

**Comment by uureel** [comment]
nope overlay is not used by the installer/updater

**Comment by paronowitz** [comment]
The only changes I made from that perspective is swapping out the boot up images/videos but again I tried the same thing using a stock unmodified x86_64 and was able to replicate the issue. I havent tried v39 yet. Im fairly happy with the system config at this point so I dont really need this working (as well as I can use ftp if I really need to transfer any new material). I may try a few other scenarious with a base image just to satisfy my curiosity.
